.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
+  - "4.0"
```

Comment:
Far enough, I had intentionally missed off 5 because this branch was originally to migrate to 4 and was going to do 5 separately afterwards. Happy to include it all in one if people think this makes more sense?